### DISCUSSION TITLE
创建 VAP PNG 的工具有哪些？

### DISCUSSION THREAD

**Discussion by cdigit** [Tencent/vap#292] [Q&A] [answered]
我是 VAP 的新手，想了解用于创建 VAP 动画的一些常用工具。 我知道我需要一个可以将动画帧导出为一堆 PNG 的工具。 我只是好奇人们用什么来实现这一目标。 我确信 Adobe AfterEffects 可能是其中一种工具，但是还有其他可以使用的工具吗？

**Comment by hexleo** [comment, marked as answer]
VAP工具只有这：https://github.com/Tencent/vap/tree/master/tool

只接受png图片输入生成vap素材，具体用什么工具生成png没有限制

**Reply by cdigit** [reply]
Thanks
